src/rez/resolved_context.py

```
@@ -1409,8 +1409,13 @@ def _minor_heading(txt):
                 try:
                     executor.execute_code(commands.source)
                 except error_class as e:
-                    msg = "Error in commands in package %r:\n%s" % (pkg.uri, str(e))
-                    raise PackageCommandError(msg)
+                    try:
+                        if (type(e) == IndentationError):
```

ouch - this will cause non-indentation error to be silently skipped